systemtests/scripts/functions

```diff
@@ -977,6 +977,33 @@ case $1 in
 esac
 }
 
+expect_grep()
+{
+
+local usage="Usage: ${FUNCNAME[0]} <expected_result> <log_file> [error message]\n    Find expected_result in log_file. Optionnaly display custom error_message if not found."
+local expected_result="$1"
+local target_file="$2"
+local error_message="$3"
+
+if [ $# -lt 2  ] || [ $# -gt 3  ]; then
+    echo -e "${FUNCNAME[0]} : wrong number of parameters.\n${usage}"
+    exit 1
+fi
+
+if [ ! -f ${target_file} ]; then
+    echo "\"${target_file}\" does not exist or is not a file."
+    exit 1
+fi
+
+if ! grep -q "${expected_result}" "${target_file}"; then
+  echo "Expected line \"${expected_result}\" was not found in log file \"${target_file}\"." >&2
+  estat=1
+  if [ "${error_message}" != "" ]; then
+    echo "-->${error_message}" >&2
```

Comment:
If running in jenkins, we probably want to print out the whole log file in the error case to be able to understand what is going on. I think we can detect if running in jenkins by checking if JENKINS_HOME is set.

Reply:
Everything is usually displayed on Jenkins, so adding the content of the file would be duplicating the output even more. 
Adding the expected line at least would help where to look for in the output.